pkg/build/graph/nodes/types.go

```diff
@@ -70,7 +70,7 @@ func BuildNodeName(o *buildapi.Build) osgraph.UniqueName {
 
 type BuildNode struct {
 	osgraph.Node
-	Build *buildapi.Build
+	*buildapi.Build
```

Comment:
> Don't feel strong about it, I just mimic what we do elsewhere.

I'd prefer to leave it like this and change the others to match this one as we touch them.